ReactWindows/ReactNative.Net46/Modules/Storage/AynceStorageModule.cs

```diff
@@ -0,0 +1,493 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using PCLStorage;
+
+namespace ReactNative.Modules.Storage
+{
+    class AsyncStorageModule : NativeModuleBase, ILifecycleEventListener
```

Comment:
This could be a perfect place for that.  I'll give that a go, and let's see what it looks like.  If it ends up being distracting, or difficult to the developer, I'll let you know.